content/announcements/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+
+Meta, designated as a [gatekeeper under the DMA](https://digital-markets-act.ec.europa.eu/gatekeepers_en), is the only company in the messaging sector subject to these new interoperability requirements, given its control over WhatsApp and Messenger. Yet, Meta’s current DMA proposal falls short of the European Union’s objectives. Instead of fostering true interoperability, it risks further entrenching Meta’s dominance—strengthening its hold on user data and reinforcing network effects that stifle competition.
+
+With this open letter, we call for a course correction—one that ensures genuine competition and user choice in the messaging space, benefiting individuals, businesses, and the broader digital ecosystem.
+
+## The Role of the XMPP Standards Foundation in the Rise of XMPP
+
+As stated on the XSF website:
+
+> The XMPP Standards Foundation (also known as the XSF and formerly the Jabber Software Foundation) is an independent, nonprofit standards development organisation whose primary mission is to define open protocols for presence, instant messaging, and real-time communication and collaboration on top of the IETF’s Extensible Messaging and Presence Protocol (XMPP).
+>
+> The XSF also provides information and infrastructure to the worldwide community of Jabber/XMPP developers, service providers, and end users.
+
+### Origins and Early Development
+
+In 1999, the goal of the original designers of XMPP was to create an open specification that would enable the emergence of a fully interoperable, federated messaging network. These designers took a pragmatic approach, ensuring that the protocol supported a transition period by supporting server-side bridging. This gateway feature enabled users to merge their legacy communication accounts—at the time primarily ICQ, Yahoo! Messenger, and Microsoft Messenger—into XMPP (then still known as Jabber) as a pivot format. The development and the maintaince of the gateways would happen on the server to adapt to quick change of protocol, but also to ensure the client could use multiple accounts in the most efficient way.
+
+This [Jabber initiative was a success](https://xmpp.org/about/history/). Developers from around the world contributed to making messaging interoperability a reality. As a result, multiple compliant servers and clients were developed, many of them as Open Source software, leading to the emergence of a vast federated network spanning thousands of domains.
+
+### Standardization and Industry Adoption
+
+XMPP was submitted to the **Internet Engineering Task Force (IETF)** and officially became a standard in 2004 under the XMPP name with the publication of **RFC 3920 and RFC 3921**.
+
+Several internet service providers across Europe, including **Libertysurf, SAPO, and 1&1**, deployed XMPP-based clients and services. The protocol gained further momentum in 2005 when **Google Talk** adopted XMPP in a federated setup, enabling large-scale messaging interoperability.
```

Comment:
[...] interoperability, including supporting groupchat, voice and video calling, and more over fully open federation.